### ISSUE TITLE
How to configure Turbine with Hystrix.NET?

### ISSUE THREAD

**Issue by mynkow** [Elders/Hystrix.NET#3]
@Sinstraliz please create a guide how to configure Turbine and capture multiple streams which are displayed at Hystrix dashboard

**Comment by kolbis** [comment]
Some more question about setting up turbine.
If I have 2 instances running the same service. they both need to report the stream.
I need to see them both aggregated in the turbine.

So, what would be the address they will report to?
what would be the address I will put in the dashboard?
how do I define custom cluster?

**Comment by mynkow** [comment]
All streams should report to Turbine and you will add Turbine stream in the dashboard.

**Comment by kolbis** [comment]
Can you please provide an example?

**Comment by Sinstraliz** [comment]
Here is the documentation for Turbine: https://github.com/Netflix/Turbine/wiki/Getting-Started-(1.x)

**Comment by sdluxeon** [comment]
Do you have any troubles with Trubine?

**Comment by kolbis** [comment]
Yes.
The link provided explain how Netflix is running it on its JAVA port.
Can you share a real example on how this will be done using .NET port?
What do we need to do code wise? What do we need to do configuration wise?

**Comment by mynkow** [comment]
I am not aware of a .NET port of Turbine. Just start an instance of turbine and publish the stats there using the streamer.

**Comment by Sinstraliz** [comment]
We were using the Netflix implementation. There is no .NET port that we know of but we did not have any issue with their version. We had a proof of concept of running Turbine in Docker container and two instances of a test app were pushing statistics to it and it was running perfectly. If you don't have a way of running JAVA applications then you will have to port it. Otherwise we just followed the documentation that they have about Turbine and we had it working. Sadly I can't find the docker file any more so I can't provide you an example but I don't remember having issues with running Turbine. If you have issues running Turbine I can try running it again on my machine and provide you with detailed steps how I set it up.
